### PR TITLE
source-paypal-transaction: ensure inline schemas, updated cdk, poetry (where possible)

### PR DESCRIPTION
**Pull request by bleonard**
This was created from a set of automated scripts. In each case, not every update was needed for every connector, but overall here is what happened:

 * `auto-schema update -s all description`: added descriptions to json and inline schemas
 * `connector-code migrate_to_yaml -c all --type source`: migrates json schemas to connectors with a manifest
 * `airbyte-ci connectors --name=<all modified connectors from above> migrate_to_base_image`: makes sure that each is using a base docker image and updates docs
 * `airbyte-ci connectors --name=<all modified connectors from above> migrate-to-poetry`: moves connectors not already using poetry to do so and updates documentation
 * `airbyte-ci connectors --name=<all modified connectors from above> up_to_date`: updated the CDK to newer (0.80.0) version

The version number and changelogs were also bumped using the `connector-code pulls` command.

### CODE REVIEW

**Review by natikgadzhi — APPROVED** [review]
LGTM. Connector version increment check freaks out, but this is fine. Good to merge in the morning.

**Comment by bleonard** [comment]
Seems likely there were other updates on the master branch. Have to mess around with the version probably.

**Review by bazarnov — APPROVED** [review]

**Comment by bazarnov** [comment]
@bleonard I was able to bump the CDK version last week for the `source-paypal`, re-base and merge with master, should resolve the conflicts.

**Comment by bleonard** [comment]
rebased!

File: airbyte-integrations/connectors/source-paypal-transaction/pyproject.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ include = "source_paypal_transaction"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "^0"
+airbyte-cdk = "0.82.0"
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-paypal-transaction/pyproject.toml:20]: Why would we pin the versions again?

**Review by bazarnov — CHANGES_REQUESTED** [review]

**Review by lazebnyi — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Pull request merged**